### DISCUSSION TITLE
怎么隐藏electron【Help】的顶部按钮

### DISCUSSION THREAD

**Discussion by terrihonkai** [opensumi/core#2735] [General]
想直接隐藏整个【Help】包括devtools和重载页面，ide-electron项目

**Comment by bytemain** [comment]
已经在官方文档中补充相应章节：https://opensumi.com/zh/docs/integrate/universal-integrate-case/custom-menu#%E5%8F%8D%E6%B3%A8%E5%86%8C%E8%8F%9C%E5%8D%95%E6%88%96%E8%80%85%E8%8F%9C%E5%8D%95%E9%A1%B9
